leap/leap_test.rb

```diff
@@ -15,26 +17,38 @@ def leap?
 
 class YearTest < Minitest::Test
   def test_leap_year
-    assert Year.leap?(1996), 'Yes, 1996 is a leap year'
+    assert Year.leap?(1996)
```

Comment:
We lose the friendly error messages. Would it be worth adding them to the leap.json file over in x-common? I don't know whether or not they're useful to people, though they are a good practice, and I like modeling good practices where possible.